docs/content/concepts/assets/asset-selection-syntax.mdx

```diff
@@ -1,32 +1,313 @@
 ---
-title: Asset Selection Syntax | Dagster
-description: To specify an asset selection, Dagster supports a simple query syntax.
+title: "Asset selection syntax | Dagster Docs"
+description: "Learn how to use Dagster's query syntax to select assets in your code, the Dagster UI, and the Dagster CLI."
 ---
 
-# Asset Selection Syntax
+# Asset selection syntax
 
-To specify an asset selection as a string, Dagster supports a simple query syntax. This selection syntax is accepted in a few different places:
+Using a simple query syntax, you can specify an asset selection as a string. In this guide, we'll cover where the syntax is supported, how it works, and some examples.
 
-- The `list` and `materialize` commands in the [asset command-line interface](/\_apidocs/cli#dagster-asset).
-- The asset filter text box on the asset graph page, in the UI.
-- The `selection` parameter of <PyObject object="define_asset_job" />. (This parameter alternatively accepts an <PyObject object="AssetSelection"/> object, which supports more complex selections built from compositions of Python objects.)
+---
+
+## Supported locations
+
+The asset selection syntax can be used in:
+
+- **In the `selection` parameter of <PyObject object="define_asset_job" />.** Alternatively, this parameter accepts an <PyObject object="AssetSelection"/> object, which supports more complex selections built from compositions of Python objects.
+
+- **The [asset command-line interface](/\_apidocs/cli#dagster-asset)** with the `list` and `materialize` commands. For example:
+
+  ```shell
+  dagster asset list --select TODO
+  dagster asset materialize --select TODO
+  ```
```

Comment:
Neither the CLI nor the `selection` parameter listed here worked with the syntax as its described here. Am I missing something?

For example, none of these worked for the CLI:

```
dagster asset list -m dagster_university --select "++manhattan_stats"
dagster asset list -m dagster_university --select ++manhattan_stats 
dagster asset list -m dagster_university --select *manhattan_stats
```

Additionally, using the syntax as described here in the `selection` parameter resulted in errors.

Reply:
@jamiedemaria D'you have any insight on this one?

Reply:
Nvm I figured it out!